test/e2e/suite/cases/stress.go

```diff
@@ -122,40 +127,40 @@ var _ = framework.CasesDescribe("Normal CSI behaviour", func() {
 		}
 
 		// Create random pods
-		wg := new(sync.WaitGroup)
-		wg.Add(len(pods))
 		for i := range pods {
-			go createPod(wg, f, i, pods)
+			createPod(f, i, pods)
 		}
-		wg.Wait()
 
 		// Wait for all the pods to become ready
-		wg.Add(len(pods))
 		for i := range pods {
-			go waitForPodToBecomeReady(wg, f, i, pods)
+			waitForPodToBecomeReady(f, i, pods)
 		}
-		wg.Wait()
 
 		// List all certificate requests that should be ready
 		crs, err := f.CertManagerClientSet.CertmanagerV1alpha2().CertificateRequests(f.Namespace.Name).List(metav1.ListOptions{})
 		Expect(err).NotTo(HaveOccurred())
 
 		// Ensure the pods volumes spec match CertificateRequest spec and the key
 		// and cert match both in pod, and on host.
-		wg.Add(len(pods))
 		for i, pod := range pods {
-			go testPod(wg, f, i, crs.Items, pod)
+			testPod(f, i, crs.Items, pod)
 		}
-		wg.Wait()
```

Comment:
Looks like you no-longer launch sets of pods in parallel. Why?

Reply:
It was actually making everything a lot slower... Doing everything sequentially made it about 25% faster believe it or not..